plugins/content/iwcsocialmeta/language/en-GB/en-GB.plg_content_iwcsocialmeta.ini

```diff
@@ -0,0 +1,18 @@
+PLG_CONTENT_IWCSOCIALMETA_XML_DESCRIPTION        = "This plugin will help to set custom meta tags for social media like Facebook,twitter etc."
+PLG_CONTENT_IWCSOCIALMETA                        = "Insta Web Care - Social Meta tag manager"
+PLG_CONTENT_IWCSOCIALMETA_LINK_OPTIONS           = "Social Media meta tags"
+
+PLG_CONTENT_IWCSOCIALMETA_ITEM_FIELD_TITLE_LABEL = "Meta Title"
+PLG_CONTENT_IWCSOCIALMETA_ITEM_FIELD_TITLE_DESC  = "Title that will send as social share title"
+
+PLG_CONTENT_IWCSOCIALMETA_DESC_ITEM_FIELD_LABEL  = "Meta Description"
+PLG_CONTENT_IWCSOCIALMETA_DESC_ITEM_FIELD_DESC   = "Description that will set as social share desc"
+
+PLG_CONTENT_IWCSOCIALMETA_TYPE_ITEM_FIELD_LABEL  = "Meta Type"
+PLG_CONTENT_IWCSOCIALMETA_TYPE_ITEM_FIELD_DESC   = "Type that will set as social share type"
+
+PLG_CONTENT_IWCSOCIALMETA_IMAGE_ITEM_FIELD_LABEL = "Meta Image"
+PLG_CONTENT_IWCSOCIALMETA_IMAGE_ITEM_FIELD_DESC  = "Image that will set as social share type"
+
+PLG_CONTENT_IWCSOCIALMETA_ALLOWED_LABLE          = "Select allowed social media"
+PLG_CONTENT_IWCSOCIALMETA_ALLOWED_DESC           = "For Selected social media meta tag will be generated"
```

Comment:
No needs to do alignment of `=` here. For example:
**Instead of**
```
PLG_CONTENT_IWCSOCIALMETA_ALLOWED_LABLE          = "Select allowed social media"
```
**Should be**

```
PLG_CONTENT_IWCSOCIALMETA_ALLOWED_LABLE="Select allowed social media"
```
for all.